update/files/multiFileWriter.go

```diff
@@ -90,6 +90,7 @@ func (m *multiFileWriter) Write(fileName string, key string, value []byte) error
 		return err
 	}
 
+	log.Trace("export key", "key", key)
```

Comment:
"export" is enough as "key" is printed after

Reply:
done